src/Command/Ide/IdeListMineCommand.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Acquia\Cli\Command\Ide;
+
+use AcquiaCloudApi\Endpoints\Applications;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Helper\TableSeparator;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Class IdeListMineCommand.
+ */
+class IdeListMineCommand extends IdeCommandBase {
+
+  protected static $defaultName = 'ide:list-mine';
+
+  /**
+   * {inheritdoc}.
+   */
+  protected function configure() {
+    $this->setDescription('List Cloud IDEs belonging to you');
+    $this->acceptApplicationUuid();
+  }
+
+  /**
+   * @param \Symfony\Component\Console\Input\InputInterface $input
+   * @param \Symfony\Component\Console\Output\OutputInterface $output
+   *
+   * @return int 0 if everything went fine, or an exit code
+   * @throws \Exception
+   */
+  protected function execute(InputInterface $input, OutputInterface $output) {
+    $acquia_cloud_client = $this->cloudApiClientService->getClient();
+    $account_ides = $acquia_cloud_client->request('get', '/account/ides');
+    $application_resource = new Applications($acquia_cloud_client);
+
+    if (count($account_ides)) {
+      $table = new Table($output);
+      $table->setStyle('borderless');
+      $table->setHeaders(['IDEs']);
+      foreach ($account_ides as $ide) {
+        $app_url_parts = explode('/', $ide->_links->application->href);
```

Comment:
This looks like it should use the same output format as `ide:info`. Can we refactor this and `ide:info` to use a common method that accepts an array of `IDE` resources and pretty-prints a table?

Maybe even use it for `ide:list` as well, with a `verbose` parameter.

Reply:
Filed CLI-530 so your feedback is not lost.